src/Rakaposhi.Core/Rakaposhi.Business.Core/DBRepository/TransTypeRepository.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Rakaposhi.Business.Core.BaseRepository;
+using Rakaposhi.Business.Core.DataObjects;
+using Rakaposhi.Data;
+using System.Data.Common;
+using System.Text.Json;
+
+namespace Rakaposhi.Business.Core.DBRepository
+{
+    public class TransTypeRepository : ITransTypeRepository
+    {
+        private IDatabase _db;
+
+        public TransTypeRepository()
+        {
+            _db = Database.Instance;
+        }
+
+        public void Add(TransType entity)
+        {
+            DbCommand cmd = _db.CreateStoreProcedure(storeProcedureName: StoreProcds.INSERT);
+            _db.AddParameter(cmd, paramName: Params.NAME, entity.Name);
+            _db.AddParameter(cmd, paramName: Params.DESCRIPTION, entity.Description);
+            entity.RecId = Convert.ToInt64(_db.ExecuteScalar(cmd));
```

Comment:
Missing the caption of second argument in AddParam.

Reply:
fixed